rainbow-me/rainbow
# fix testnet support

I added a toggle in the network switcher since if we enable them by default things are ugly for a normal user (see video) happy to change the UX around this as I think the network switcher has no point for a normal user.

update:
- added https://faucet.paradigm.xyz for clout but also cause its the best

POW(updated): 
https://cloud.skylarbarrera.com/Screen-Recording-2021-11-27-00-43-57.mp4

QA: 
-  make sure the options are consistent on iOS and android.
-  make sure the setting persists between sessions.
-  [example.walletconnect.org](example.walletconnect.org) should show networks switching properly
- tx's should work as expected, send & walletconnect
